content/en/developers/ide_plugins/idea/live_debugger.md

```diff
@@ -2,72 +2,72 @@
 
 {{< img src="/developers/ide_plugins/idea/live_debugger/gutter-icon.png" alt="Gutter icon in the source editor" style="width:100%;" >}}
 
-Click the icon to open the Datadog tool window and select the log probe. Right-click the icon for options to enable, disable, edit, and delete the log probe. These actions are described in later sections.
+Click the icon to open the Datadog tool window and show the selected logpoint. Right-click the icon for options to enable, disable, edit, and delete the logpoint. These actions are described in later sections.
 
-## Managing log probes
+## Managing logpoints
 
-### Add a log probe
-To add a log probe, right-click on a line of code in the source editor and select **Add a Log to a Live Service**. A dialog appears where you can enter the service name, the environment, and the log message you would like to have emitted at runtime:
+### Add a logpoint
+To add a logpoint, right-click on a line of code in the source editor and select **Add a Log to a Live Service**. A dialog appears where you can enter the service name, the environment, the log message you would like to have emitted at runtime, and an optional logpoint condition expression:
 
-{{< img src="/developers/ide_plugins/idea/live_debugger/new-log-probe.png" alt="Add a new log probe" style="width:75%;" >}}
+{{< img src="/developers/ide_plugins/idea/live_debugger/new-logpoint.png" alt="Add a new log probe" style="width:75%;" >}}
 
-The log message field accepts a log template that contains descriptive text and variable references—see the [Dynamic Instrumentation expression language][3] documentation for details. The log message is generated using the runtime state immediately *prior* to the line of code being executed.  
+The log message field accepts a log template that contains descriptive text and variable references—see the [Dynamic Instrumentation expression language][3] documentation for details. The log message is generated using the runtime state immediately *prior* to the line of code being executed.  If a condition is defined, log events will be generated only when the condition evaluates to true.
 
-Log probes expire automatically after 60 minutes.
+Logpoints expire automatically after 60 minutes.
 
 #### Local and remote versions
-Notice that the remote code may be a different version compared to the source code in the IDE. The New Log Probe dialog displays the version of the code that is deployed remotely, if possible, so that you can see exactly where the log probe will be placed. This requires that your application or service is [tagged with Git information][4].
+Notice that the remote code may be a different version compared to the source code in the IDE. The New Logpoint dialog displays the version of the code that is deployed remotely, if possible, so that you can see exactly where the logpoint will be placed. This requires that your application or service is [tagged with Git information][4].
```

Comment:
```suggestion
Notice that the remote code may be a different version compared to the source code in the IDE. The New Logpoint dialog displays the version of the code that is deployed remotely, if possible, so that you can see exactly where the logpoint is being placed. This requires that your application or service is [tagged with Git information][4].
```